### PR TITLE
Update rapids-cmake version for pyraft in update-version.sh

### PR DESCRIPTION
**Pull request by vyasr**
Previously only pylibraft's copy was being updated, leading to an out-of-date branch of rapids-cmake being pulled for pyraft.

### CODE REVIEW

**Review by sevagh — APPROVED** [review]

**Review by robertmaynard — CHANGES_REQUESTED** [review]
As we ae now up to 3 places using rapids-cmake we should switch over to the approach cudf is using where we have a single `fetch_rapids` that each project includes. 

https://github.com/rapidsai/cudf/blob/branch-22.08/fetch_rapids.cmake

File: python/pylibraft/CMakeLists.txt [hunk 1]
```diff
@@ -16,9 +16,7 @@ cmake_minimum_required(VERSION 3.20.1 FATAL_ERROR)
 
 set(pylibraft_version 22.08.00)
 
-file(DOWNLOAD https://raw.githubusercontent.com/rapidsai/rapids-cmake/branch-22.08/RAPIDS.cmake
-        ${CMAKE_BINARY_DIR}/RAPIDS.cmake)
-include(${CMAKE_BINARY_DIR}/RAPIDS.cmake)
+include(../fetch_rapids.cmake)
```
Comment (robertmaynard) [inline python/pylibraft/CMakeLists.txt:19]: up two directories for `fetch_rapids.cmake`

File: python/raft/CMakeLists.txt [hunk 2]
```diff
@@ -16,9 +16,7 @@ cmake_minimum_required(VERSION 3.20.1 FATAL_ERROR)
 
 set(pyraft_version 22.08.00)
 
-file(DOWNLOAD https://raw.githubusercontent.com/rapidsai/rapids-cmake/branch-22.06/RAPIDS.cmake
-        ${CMAKE_BINARY_DIR}/RAPIDS.cmake)
-include(${CMAKE_BINARY_DIR}/RAPIDS.cmake)
+include(../fetch_rapids.cmake)
```
Comment (robertmaynard) [inline python/raft/CMakeLists.txt:19]: up two directories for `fetch_rapids.cmake`

**Review by robertmaynard — CHANGES_REQUESTED** [review]

**Comment by vyasr** [comment]
Whoops sorry about that.

**Review by robertmaynard — APPROVED** [review]

**Comment by cjnolet** [comment]
@gpucibot merge

**Pull request merged**